modular_skyrat/modules/sec_haul/code/misc/vending.dm

```diff
@@ -20,7 +20,6 @@
 	contraband = list(
 		/obj/item/clothing/glasses/sunglasses = 2,
 		/obj/item/storage/fancy/donut_box = 2,
-		/obj/item/armament_token/sidearm_blackmarket = 2,
```

Comment:
actually, before merge, I *will* say that these had a use

Reply:
Damn shame